error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
Diameter: 7.0
TIME:  0.152545928955
CLOCKTIME:  3.811177
	Command being timed: "python /home/cpennycu/networkx/gt.py -f /home/cpennycu/gdata/sf/sf-4-1000-2.3.snap -d 0"
	User time (seconds): 5.79
	System time (seconds): 0.33
	Percent of CPU this job got: 570%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:01.07
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 63812
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 30518
	Voluntary context switches: 209
	Involuntary context switches: 272
	Swaps: 0
	File system inputs: 56
	File system outputs: 16
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
